eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionWriterTest.java

```diff
@@ -35,6 +35,7 @@
 import uk.ac.ebi.eva.accession.core.SubmittedVariant;
 import uk.ac.ebi.eva.accession.core.SubmittedVariantAccessioningService;
 import uk.ac.ebi.eva.accession.core.configuration.SubmittedVariantAccessioningConfiguration;
+import uk.ac.ebi.eva.accession.core.io.FastaSequenceReader;
```

Comment:
this import is unused. applies to other tests

Reply:
`FastaSequenceReader` is used in the `@Before` method of the class